src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -12,14 +14,18 @@ import { WorkItemService } from '../work-item.service';
   templateUrl: './work-item-detail.component.html',
   styleUrls: ['./work-item-detail.component.scss']
 })
-export class WorkItemDetailComponent implements OnInit {
+
```

Comment:
Why was this new line inserted? I think the main style we and the Angluar team follow is to have the @component directly over the class it is modifying.